dotnet/src/dotnetframework/GxClasses/Model/GXWebProcedure.cs

```diff
@@ -115,18 +116,20 @@ protected override void sendCacheHeaders()
 
 		private void setOuputFileName()
 		{
-			string fileName = GetType().Name;
-			string fileType = "pdf";
-			if (!string.IsNullOrEmpty(outputFileName))
+			if (fileContentInline)
 			{
-				fileName = outputFileName;
-			}
-			if (!string.IsNullOrEmpty(outputType))
-			{
-				fileType = outputType.ToLower();
+				string fileName = GetType().Name;
+				string fileType = "pdf";
+				if (!string.IsNullOrEmpty(outputFileName))
+				{
+					fileName = outputFileName;
```

Comment:
This code is about PDF Procedures. Maybe it mixed up from another branch?

Reply:
You are right. Moved this to #826.
I found the error about sending an unnecessary content-disposition header while testing this feature, but it is a separate issue.